content/03.results.md

```diff
@@ -100,16 +114,22 @@ Genotype information is not available in the SVPOP catalog hence genotyping perf
-The second graph (in the following called *cactus graph*) was derived from a multiple genome alignment of the five strains using our Cactus tool [@doi:10.1101/gr.123356.111].
-The *VCF graph* is mainly linear and highly dependent on the reference genome.
-In contrast, the *cactus graph* is structurally complex and free of reference bias.
-
+This bypasses the need for generating an explicit variant catalog relative to a linear reference, which could be a source of error (for example, from reference bias during read mapping and variant calling).
+Furthermore, genome alignments from graph-based software such as Cactus [@doi:10.1101/gr.123356.111] can contain complex structural variation that is extremely difficult to represent, let alone call, outside of a graph.
+We therefore investigated whether genome graphs derived from alignments of de-novo assemblies yield advantages for SV genotyping.
+
+[//]: # (The description of the experimental set up in this paragraph is unclear to me. I've tried to clean it up, but I'm not totally sure I understood it well enough to do it accurately.)
+We analyzed public sequencing datasets for 12 yeast strains from two related clades (*S. cerevisiae* and *S. paradoxus*) [@doi:10.1038/ng.3847].
+We compared genotyping results using two different types of genome graphs.
+The graphs were constructed using 5 of the 12 strains.
+*S.c. S288C* was used as the reference strain, and we selected two other strains from each of the two clades (see [Methods](#yeast-graph-analysis)). 
+The first graph (called *VCF graph* below) was constructed by calling SVs in the VCF relative to the *S.c. S288C* and then adding these SVs onto it. 
+The SVs were identified using three methods: Assemblytics [@doi:10.1093/bioinformatics/btw369], AsmVar [@doi:10.1186/s13742-015-0103-4] and paftools [@doi:10.1093/bioinformatics/bty191].
+The second graph (called *cactus graph* below) was derived from a multiple genome alignment of the five strains using our Cactus tool [@doi:10.1101/gr.123356.111].
+The *VCF graph* is mostly linear and highly dependent on the reference genome.
+In contrast, the *cactus graph* is structurally complex and relatively free of reference bias.
+
+[//]: # (I think this mapping comparison could probably be moved to the supplement. The rest of the main text is very focused on SV typing. I personally find it a bit confusing when I suddenly start seeing mapping results. It's good that we thought about this, but perhaps we could just mention the headline results in a sentence and refer people to the supplement. What do you guys think? I also think that we could do away with the MAPQ plot. To my mind, identity is a better measure for what we're trying to show, and it seems to make the point even more strongly.)
```

Comment:
I think MAPQs worth leaving in somewhere.  It provides some evidence that, overall, the increased identity is not coming at the cost of added ambiguity.

Reply:
Like as some way of penalizing the graph complexity? That makes some sense to me. If that's the justification though, then I think we need to make it explicit that it's what we're trying to do. The motivation was not clear to me. I also would recommend switching the order of the identity and the MAPQ results, and emphasizing the identity results in the text rather than the MAPQ results. To my mind, the logical flow would be to first show that we can obtain mappings with higher identity and then to show that it wasn't simply because of a more complex graph.